backend/db/mongo.go

```diff
@@ -188,7 +188,7 @@ func DecodeBsonData(data bson.M) (models.Game, error) {
 	game := models.Game{
 		Id:           data["_id"].(primitive.ObjectID).Hex(),
 		Name:         data["name"].(string),
-		Author:       data["author"].(string),
+		Developer:    data["author"].(string),
```

Comment:
I believe the key string will need to be "developer". Run the tests to make sure.

Reply:
It passes the tests when creating the pull request... are there additional tests we haven't automated?

Reply:
Yep, the database_test.go doesn't run with the automation. If you look at the automated build tests you can see:

Run cd backend && go test -v .
github.com/jak103/usu-gdsf	[no test files]

I have my IDE setup, so I'm not 100% on this, but I believe the command is "go test github.com/jak103/usu-gdsf/backend/db"

Reply:
I'm running into issues with the .env file that I will have to work through, that command seems like it would be testing the main repository, though, right?

Reply:
Yeah, so the command might be something like "go test ./backend/db/"?

Reply:
I'm trying to run the tests but getting invalid DbType even though my .env appears to be configured correctly. Might need to look back at whatever was said when we implemented the .env file.

Reply:
Ah, we'll have to figure it out in class